Sources/SwiftParser/generated/Parser+TokenSpecSet.swift

```diff
@@ -0,0 +1,2027 @@
+      case .mutableAddressWithOwner:
+        return .keyword(.mutableAddressWithOwner)
+      case .mutableAddressWithNativeOwner:
+        return .keyword(.mutableAddressWithNativeOwner)
+      case ._read:
+        return .keyword(._read)
+      case ._modify:
+        return .keyword(._modify)
+      case .`init`:
+        return .keyword(.`init`)
+      }
+    }
+  }
+  
+  enum AsExprQuestionOrExclamationMark: TokenSpecSet {
+    case postfixQuestionMark
+    case exclamationMark
+    
+    init?(lexeme: Lexer.Lexeme) {
+      switch PrepareForKeywordMatch(lexeme) {
+      case TokenSpec(.postfixQuestionMark):
+        self = .postfixQuestionMark
+      case TokenSpec(.exclamationMark):
+        self = .exclamationMark
+      default:
+        return nil
+      }
+    }
+    
+    var spec: TokenSpec {
+      switch self {
+      case .postfixQuestionMark:
+        return .postfixQuestionMark
+      case .exclamationMark:
+        return .exclamationMark
+      }
+    }
+  }
+  
+  enum AttributedTypeSpecifier: TokenSpecSet {
+    case `inout`
+    case __shared
+    case __owned
+    case isolated
+    case _const
+    case borrowing
+    case consuming
+    
+    init?(lexeme: Lexer.Lexeme) {
+      switch PrepareForKeywordMatch(lexeme) {
+      case TokenSpec(.`inout`):
+        self = .`inout`
+      case TokenSpec(.__shared):
+        self = .__shared
+      case TokenSpec(.__owned):
+        self = .__owned
+      case TokenSpec(.isolated):
+        self = .isolated
+      case TokenSpec(._const):
+        self = ._const
+      case TokenSpec(.borrowing):
+        self = .borrowing
+      case TokenSpec(.consuming):
+        self = .consuming
+      default:
+        return nil
+      }
+    }
+    
+    var spec: TokenSpec {
+      switch self {
+      case .`inout`:
+        return .keyword(.`inout`)
+      case .__shared:
+        return .keyword(.__shared)
+      case .__owned:
+        return .keyword(.__owned)
+      case .isolated:
+        return .keyword(.isolated)
+      case ._const:
+        return .keyword(._const)
+      case .borrowing:
+        return .keyword(.borrowing)
+      case .consuming:
+        return .keyword(.consuming)
+      }
+    }
+  }
+  
+  enum AvailabilityConditionAvailabilityKeyword: TokenSpecSet {
+    case poundAvailable
+    case poundUnavailable
+    
+    init?(lexeme: Lexer.Lexeme) {
+      switch PrepareForKeywordMatch(lexeme) {
+      case TokenSpec(.poundAvailable):
+        self = .poundAvailable
+      case TokenSpec(.poundUnavailable):
+        self = .poundUnavailable
+      default:
+        return nil
+      }
+    }
+    
+    var spec: TokenSpec {
+      switch self {
+      case .poundAvailable:
+        return .poundAvailable
+      case .poundUnavailable:
+        return .poundUnavailable
+      }
+    }
+  }
+  
+  enum AvailabilityLabeledArgumentLabel: TokenSpecSet {
+    case message
+    case renamed
+    case introduced
+    case obsoleted
+    case deprecated
```

Comment:
I found that this doesn't contain the same as in `AvailabilityArgumentKind`

It have also a special handling

https://github.com/apple/swift-syntax/blob/5474478ccf6bd3319c5c3564d2eed15dc3e45eac/Sources/SwiftParser/Availability.swift#L57-L96

Reply:
These are exactly the interesting cases. A short-term solution would probably be to define the case from `AvailabilityArgumentKind` that aren’t in `AvailabilityLabeledArgumentLabel` in a separate `TokenSpecSet` and then use `self.at(anyIn:or:)` in Availability.swift:104. In general for these cases, we might need to consider if there is a bug in the parser or if we can model the syntax tree better so that the generated `TokenSpecSet` matches the parser.